### PR TITLE
chore/issue template modification for v2

### PR DESCRIPTION
**Pull request by CommanderStorm**
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]:
- [x] I have read and understand the pull request rules.

# Description

As discussed in #4171 some modifications to our issue template are likely neccesary before launching v2.0
This PR is the discussion for this modification

## Type of change

Please delete any options that are not relevant.

- Other

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [x] I have commented my code, particularly in hard-to-understand areas (including JSDoc for methods)
- [x] My changes generates no new warnings

### CODE REVIEW

**Comment by louislam** [comment]
Is it possible to enable "Issues" in your fork?

**Comment by CommanderStorm** [comment]
> Is it possible to enable "Issues" in your fork? So that I can preview it.

I have changed the repo settings: https://github.com/CommanderStorm/uptime-kuma/issues/new/choose

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -73,22 +81,23 @@ body:
...
-      required: false
-  - type: input
-    id: nodejs-version
+  - type: textarea
+    id: deployment-info
     attributes:
-      label: "🟩 NodeJS Version"
-      description: "If running with Node.js? which version are you running?"
-      placeholder: "Ex. 14.18.0"
+      label: "🖥️ The system is deployed as such"
```
Comment (chakflying) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:87]:
```suggestion
      label: "🖥️ Deployment Environment"
```

**Review by chakflying — COMMENTED** [review]
It's a bit weird that `related-issues` and `no-duplicate-issues` are both marked as required. I think would make sense to fill out either one or the other.

Maybe we can merge them, just use the textarea and say "or explain that you are unable to find any related issues".

**Comment by chakflying** [comment]
Also, I think GitHub has a bug as the inline editor would overlay "leave a comment" on top of the placeholder. I'm using Firefox on Windows.

image

**Comment by louislam** [comment]
Chrome too

image

**Comment by CommanderStorm** [comment]
I don't know what would be the best way to go around this. This seems to be a GitHub bug.
Upvoting https://github.com/orgs/community/discussions/76763 may help.

I could make all `placeholder`'s into pre-filled values, but users would likely not remove that..

File: .github/ISSUE_TEMPLATE/ask-for-help.yaml [hunk 2]
```diff
@@ -56,19 +56,20 @@ body:
...
-      required: false
-  - type: input
-    id: nodejs-version
+  - type: textarea
+    id: deployment-info
     attributes:
-      label: "🟩 NodeJS Version"
-      description: "If running with Node.js? which version are you running?"
-      placeholder: "Ex. 14.18.0"
+      label: "🖥️ The system is deployed as such"
```
Comment (chakflying) [inline .github/ISSUE_TEMPLATE/ask-for-help.yaml:62]:
```suggestion
      label: "🖥️ Deployment Environment"
```

**Review by chakflying — COMMENTED** [review]
Consistency with the Bug Report Template.

File: .github/ISSUE_TEMPLATE/ask-for-help.yaml (hunk 2 above)
Comment (CommanderStorm) [reply .github/ISSUE_TEMPLATE/ask-for-help.yaml:62]: Fair enough. did not think about this

**Comment by louislam** [comment]
> https://github.com/orgs/community/discussions/76763

It seems that they fixed the issue, it can be merged now.

**Pull request merged**